M/muesli/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "muesli"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://bitbucket.org/ignromero/muesli.git", "27e8204971602cb042d633b8b5f87761272b10df")
+    DirectorySource(joinpath(@__DIR__, "bundled"))
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/muesli
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cmakesupport.patch
+
+mkdir build &&  cd build && \
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target install --parallel "$(nproc)" 
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
Instead of listing all platforms you could say `platforms = supported_platforms()`, and then use `filter` to remove Windows. Currently you're excluding FreeBSD (presumably accidentally).